src/docgen/transpile/transpile.ts

```diff
@@ -374,7 +374,7 @@ export interface TranspiledModuleLike {
  */
 export interface Transpile {
   /**
-   * The language of the tranpiler.
+   * The language of the transpiler.
    */
   language: string;
```

Comment:
Should this property be readonly? (Although unrelated to the present change)

```suggestion
  readonly language: string;
```

Reply:
Done